Portfolio.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+	<html>
+		<head>
+			<link rel="stylesheet" type="text/css" href="css/styles.css">
+		</head>
+			<body>
+				<nav class="topNav">
+					<a href="index.html">About Me</a> | <i>Portfolio</i>
+				</nav>
+
+				<h1>Joe Johnson</h1>
+
+				<hr>
+
+				<h2>Experience</h2>
+				
+				<p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an <b>awesome</b> Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.</p>
+
+				<h3>Portfolio</h3>
+					<ul>
+						<li><b>Web Developer</b>, Relaxr 2015 - present<br>
+Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</li>
+						<img src="images/relaxr_deliverable.png" alt="website example" title="relaxr"/>
+
+						<li><b>Web Developer</b>, Startup Matchmaker 2015 - present<br>
+Used a design team's wireframes to develop this company's responsive homepage.</li>
+						<img src="images/startup_matchmaker_deliverable.png"/>
+
+						<li><b>Web Developer</b>, CitiPix 2015 - present<br>
+Built web app prototype that allows users to store and quickly retrieve photos using keywords.</li>
+
+						<img src="images/citipix_deliverable.png"/>
+					
+					</ul>
+
+				<h3>Education</h3>
+
+					<p>General Assembly - FEWD <i>2015</i></p>
```

Comment:
What is the better tags to use for lists. The heading is Education and you have a list of schools